### ISSUE TITLE
.desktop file generation

### ISSUE THREAD

**Issue by freswa** [ncopa/xfce-polkit#2]
make generates the following line in the desktop file

```
Exec=/usr/libexec/xfce-polkit
```

Which should be:

```
Exec=/usr/lib/xfce-polkit/xfce-polkit
```

Any ideas?

**Comment by ncopa** [comment]
What happens if you do: `./configure --libexecdir=/usr/lib/xfce-polkit`

**Comment by rapgro** [comment]
An user of Fedora reports:
The problem is (and that's the report) the (null) output inside the xfce4-session's information tab. I would have expected a bit more descriptive information there. So I assume, that if (null) is being shown under Fedora 22 then it most likely will show (null) under Fedora RH as well. May I ask if you could have a look in your xfce4-sessions and report if it shows (null) for you as well ?

Downstream bug:
https://bugzilla.redhat.com/show_bug.cgi?id=1281464

**Comment by freswa** [comment]
@ncopa Your hint works. So what about those underscores in front of "Comment" and "Name"?

**Comment by rapgro** [comment]
Those underscores prevent a descriptive title in the autostart settings dialog of xfce4-session.

**Comment by rdieter** [comment]
and .desktop files should generally have valid Name (and ideally Comment) keys.  To put another way, why use non-standard _Name and _Comment here rather than Name and Comment?

**Comment by ncopa** [comment]
i guess this was fixed with #3
